lib/events.rb

```diff
@@ -11,6 +11,10 @@
     Organization.update_from_json! payload.deep_symbolize_keys[:organization]
   end
 
+  event :InvitationCreated do |payload|
+    Invitation.create payload.deep_symbolize_keys[:invitation]
```

Comment:
Use `create!`

Also, delegate this method into the invitation, as `import_from_json!`, like in - most - other models.